packages/firestore/test/unit/local/query_cache.test.ts

```diff
@@ -47,6 +47,17 @@ describe('MemoryQueryCache', () => {
   });
 
   genericQueryCacheTests();
+
+  it('cannot remove nonexistent query', async () => {
+    // We don't check on indexeddb query cache, but we do for memory.
+    let gotError = false;
+    try {
+      await cache.removeQueryData(testQueryData(QUERY_ROOMS, 1, 1));
+    } catch {
+      gotError = true;
+    }
+    expect(gotError).to.be.true;
```

Comment:
So this should only be true for MemoryPersistence?

Reply:
Yes, I updated the comment.